models/DataObject/Data/ObjectMetadata.php

```diff
@@ -117,23 +117,23 @@ public function __call($name, $arguments)
     /**
      * @param $object
      * @param string $ownertype
-     * @param $ownername
-     * @param $position
-     * @param $index
+     * @param string $ownername
+     * @param int $position
```

Comment:
`$position`  can be a string as well (e.g. localized fields).

Reply:
You are right. I changed it